src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -1567,7 +1559,7 @@ void SoundEditor::modEncoderButtonAction(uint8_t whichModEncoder, bool on) {
 	}
 }
 
-bool SoundEditor::setup(Clip* clip, const MenuItem* item, int32_t sourceIndex) {
+bool SoundEditor::setup(Clip* clip, const MenuItem* item, const MenuItem* parent, int32_t sourceIndex) {
```

Comment:
should be `const Submenu*` so there's no need for the cast. Probably just need to forward declarare `class Submenu;` in the header (no need to include submenu.h there.)